### PR TITLE
Add status and change font sizes

### PR DESCRIPTION
**Pull request by ragnorc**

### CODE REVIEW

File: packages/glenstack.com/src/components/landing/footer.tsx [hunk 1]
```diff
@@ -190,9 +190,28 @@ export const Footer: FC<{ noCTA?: boolean }> = ({ noCTA = false }) => {
...
+                {/* <a href="/terms/">Terms of Service</a>{" "} */}
+              </li>
+            </ul>{" "}
+            <div className="mt-6 lg:my-0 lg:w-1/3 lg:text-right">
+              <a
+                href="https://status.glenstack.com"
+                className="inline-block lg:ml-auto"
+              >
+                <span className="inline-block w-2 h-2 mr-1 rounded-full bg-green-500"></span>{" "}
+                <span>All Systems Operational</span>
```
Comment (GregBrimble) [inline packages/glenstack.com/src/components/landing/footer.tsx:211]: Can we pull this in live?

File: packages/glenstack.com/src/components/landing/footer.tsx [hunk 2]
```diff
@@ -190,9 +190,28 @@ export const Footer: FC<{ noCTA?: boolean }> = ({ noCTA = false }) => {
...
-          <p className="text-base text-gray-400 xl:text-center">
-            &copy; 2021 Glenstack Ltd. All rights reserved.
-          </p>
+          <div className="w-full text-sm font-light text-gray-400 lg:flex lg:justify-between lg:items-center">
+            <span className="block mb-2 lg:w-1/3 lg:my-0">
+              ©2021 Glenstack Ltd. Made worldwide.
+            </span>{" "}
+            <ul className="lg:w-1/3 lg:text-center">
+              <li className="inline-block mb-0 mr-4 last:mr-0">
+                {/* <a href="/privacy/">Privacy Policy</a>{" "} */}
```
Comment (GregBrimble) [inline packages/glenstack.com/src/components/landing/footer.tsx:199]: Don't these links exist already above in the Legal section? Can we just take them out here?

**Review by GregBrimble — CHANGES_REQUESTED** [review]

File: packages/glenstack.com/src/components/landing/footer.tsx (hunk 2 above)
Comment (ragnorc) [reply packages/glenstack.com/src/components/landing/footer.tsx:199]: Forgot what I did there. Let's see 😛

File: packages/glenstack.com/src/components/landing/footer.tsx (hunk 1 above)
Comment (ragnorc) [reply packages/glenstack.com/src/components/landing/footer.tsx:211]: Does status flare provide an endpoint?

**Pull request merged**